lte/gateway/fabfile.py

```diff
@@ -310,7 +337,8 @@ def run_integ_tests(tests=None):
     command in test machine:
     $ make integ_test TESTS=s1aptests/test_attach_detach.py
     """
-    test_host = vagrant_setup("magma_test", destroy_vm=False)
+    test_host = vagrant_setup("magma_test", destroy_vm=False,
```

Comment:
what is the reason for this change?

Reply:
yes and not: it looks like vagrant_setup was defaulting `force_provision` to string `False`., which caused to be True on the comparison

I am going to change instead force_provision at vagrant_setup to use boolean value by default. 

This is the problem with types on fab: the arguments we pass through the command line always come as strings.